### PR TITLE
Fix Run argument names

### LINKED ISSUE #813: --fix-whitespace doesn't work in dotnet-format 5.0.146601
When using both `--fix-whitespace --fix-style warn`  in dotnet-format 5.0.146601 the checker doesn't report whitespace errors.

### PR DESCRIPTION
**Pull request by JoeRobich**
Fixes #813

### CODE REVIEW

File: src/Program.cs [hunk 1]
```diff
@@ -40,9 +40,9 @@ private static async Task<int> Main(string[] args)
         public static async Task<int> Run(
             string? workspace,
             bool folder,
-            bool whitespace,
-            string? style,
-            string? analyzers,
+            bool fixWhitespace,
```
Comment (jmarolf) [inline src/Program.cs:43]: so if these don't match things just silently stop working?

**Review by jmarolf — APPROVED** [review]

File: src/Program.cs (hunk 1 above)
Comment (JoeRobich) [reply src/Program.cs:43]: I am working on a test that will help us stay correct.

Comment (JoeRobich) [reply src/Program.cs:43]: Added test and delegate.

**Comment by jmarolf** [comment]
@JoeRobich looks like the test [CommandLine_AllArguments_Bind](https://dev.azure.com/dnceng/public/_build/results?buildId=821283&view=ms.vss-test-web.build-test-results-tab) is failing in CI

**Comment by JoeRobich** [comment]
> looks like the test [CommandLine_AllArguments_Bind](https://dev.azure.com/dnceng/public/_build/results?buildId=821283&view=ms.vss-test-web.build-test-results-tab) is failing in CI

@jmarolf Fixed in https://github.com/dotnet/format/pull/818

**Pull request merged**